### ISSUE TITLE
Use IETF language tags

### ISSUE THREAD

**Issue by williamli** [michaelwittig/node-i18n-iso-countries#123]
I think you can extend the current language code to use IETF language tags https://en.wikipedia.org/wiki/IETF_language_tag

en_us = US English
en_gb = UK English

zh_hk = Hong Kong Traditional Chinese
zh_tw = Taiwan Traditional Chinese
zh_cn = Mainland Simplified Chinese

**Comment by michaelwittig** [comment]
Good catch. According to https://tools.ietf.org/html/rfc5646#section-2.2.1 the primary language subtag is ISO 639-1!

Examples from https://tools.ietf.org/html/rfc5646#appendix-A
`zh-Hant` (Chinese written using the Traditional Chinese script)
`zh-Hans` (Chinese written using the Simplified Chinese script)

**Comment by williamli** [comment]
In my other projects, we use a simpler form of this (no need in your example, the Hant and Hans section is omitted).

https://en.wikipedia.org/wiki/Language_localisation <https://en.wikipedia.org/wiki/Language_localisation>

I believe most of the internet uses this form.
In Hong Kong, we have
zh-HK for our official Traditional Chinese
and
en-HK for our official English

Example for Chinese


zh-CN	Mainland China, simplified characters
zh-TW	Taiwan, traditional characters
zh-HK	Hong Kong, traditional characters

**Comment by michaelwittig** [comment]
I have no opinion if we should go with `zh-Hans` or `zh-CN`. I only want to keept it backward compatible and we should go with some "standard" to which we can refer in the docs.

Let's wait a few days and see if someone joines the discussion? What is your preferred way?

**Comment by williamli** [comment]
i prefer zh-CN, 
as I said before, this is how most other programmers handles their locale anyway.

https://github.com/umpirsky/locale-list/blob/master/data/rw/locales.php <https://github.com/umpirsky/locale-list/blob/master/data/rw/locales.php>

**Comment by piranna** [comment]
API shouldn't change, so supporting all forms is just a matter of having mapping lists...

**Comment by kepek** [comment]
👍
